.gitignore

```diff
@@ -0,0 +1,5 @@
+.idea
```

Comment:
I mean that the entire `.gitignore` file does not belong into this pull request. Let's discuss [here](https://github.com/Cryxnoob/TeamProjectTH-BingenEP-1/pull/14#issuecomment-319466219).